docs/BR.md

```diff
@@ -1326,21 +1326,32 @@ No stipulation.
 
 The following SHALL apply for communicating the status of Certificates which include an Authority Information Access extension with an id-ad-ocsp accessMethod.
 
+Authoritative OCSP responses MUST be available (i.e. the responder MUST NOT respond with the "unknown" status) starting no more than 15 minutes after the Certificate's `notAfter` timestamp.
```

Comment:
Perhaps the obligation to provide OCSP responses should be tied to when the Certificate's serial number transitions from "unassigned" to "assigned". This would accommodate any backdating and yet ensure that responses are distributed timely.

Reply:
> s/notAfter/notBefore/ perhaps?

Heh oops, thanks for catching this typo.

> Backdating the notBefore timestamp by more than 15 minutes would mean that the CA would have had to make available an authoritative OCSP response some number of minutes earlier than it possibly could have done.

This is a good point. I've always wondered why the BRs allow backdating by up to 48 hours (I've never seen a need to backdate more than 1hr, and even that feels generous).

> I think the "15 minutes" countdown should begin when the "certificate signing operation" occurs.

Yeah, I don't love this solution, but I think you're right that it's likely the best path forward. And at least the trail has been blazed somewhat by the existing backdating requirements referencing this same timestamp. I'll go this direction, thanks.